exchanges/gateio/gateio_test.go

```diff
@@ -2533,49 +2533,33 @@ func TestUpdateOrderExecutionLimits(t *testing.T) {
 	t.Parallel()
 	testexch.UpdatePairsOnce(t, g)
 
-	err := g.UpdateOrderExecutionLimits(t.Context(), 1336)
-	if !errors.Is(err, asset.ErrNotSupported) {
-		t.Fatalf("received %v, expected %v", err, asset.ErrNotSupported)
-	}
-
-	err = g.UpdateOrderExecutionLimits(t.Context(), asset.Options)
-	if !errors.Is(err, common.ErrNotYetImplemented) {
-		t.Fatalf("received %v, expected %v", err, common.ErrNotYetImplemented)
+	scenarios := []struct {
+		assetType     asset.Item
+		expectedError error
+	}{
+		{assetType: asset.Spot, expectedError: nil},
+		{assetType: asset.Futures, expectedError: nil},
```

Comment:
Need to add in checks for `asset.CoinMarginedFutures` and `asset.USDTMarginedFutures` since they've been split from `asset.Futures` now as this test is currently failing with unsupported asset type.

This also highlights that the `UpdateOrderExecutionLimits` still has a switch case check for `asset.Futures` but will never get hit because the very first `g.SupportsAsset(a)` check will fail with the above